print_nanny_webapp/templates/subscriptions/list.html

```diff
@@ -0,0 +1,249 @@
+                    <div id="card-element">
+                        <!-- A Stripe Element will be inserted here. -->
+                    </div>
+
+                    <!-- Used to display form errors. -->
+                    <div id="card-errors" role="alert"></div>
+                    <div id="card-success" role="alert">
+                        <span id="success-message"></span>
+                    </div>
+
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-light" data-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-primary" id="card-button">Submit Payment</button>
+                </div>
+            </div><!-- /.modal-content -->
+        </div><!-- /.modal-dialog -->
+    </div><!-- /.modal -->
+
+    <table id="subscription-history" class="table table-hover table-centered">
+        <thead>
+            <tr>
+                <th>Date</th>
+                <th>Price</th>
+                <th>Status</th>
+                <th>Start Date</th>
+                <th>End Date</th>
+                <th>Action</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for subscription in SUBSCRIPTIONS %}
+                <tr>
+                    <td>{{ subscription.created_datetime }}</td>
+                    <td>{{ subscription.plan.amount_float | floatformat:2 | intcomma }} {{ subscription.plan.currency | upper }}</td>
+                    <td>
+                        {% if subscription.status == "active" %}
+                            <i class="mdi mdi-circle text-success"></i>
+                        {% elif subscription.status == "unpaid" or subscription.status == "past_due" %}
+                            <i class="mdi mdi-circle text-warning"></i>
+                        {% elif subscription.status == "past_due" %}
+                            <i class="mdi mdi-circle text-dange"></i>
+                        {% else %}
+                            <i class="mdi mdi-circle"></i>
+                        {% endif %}
+                        &nbsp;{{ subscription.status | title }}
+                    </td>
+                    <td>{{ subscription.current_period_start_datetime }}</td>
+                    <td>{{ subscription.current_period_end_datetime }}</td>
+                    <td class="table-action">
+                        <a href="javascript: void(0);" class="action-icon"> <i class="mdi mdi-pencil"></i></a>
+                        {% if subscription.status == "active" or subscription.status == "unpaid" or subscription.status == "past_due" %}
+                            <a href="javascript: void(0);" class="action-icon"> <i class="mdi mdi-delete"></i></a>
+                        {% endif %}
+                    </td>
+                </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+</div>
+
+<script>
+    window.addEventListener('load', function () {
```

Comment:
Can you drop a breadcrumb back to the original source code here? Thank you! https://github.com/dj-stripe/dj-stripe/blob/stable/2.4/tests/apps/example/templates/payment_intent.html